src/manifest/representation.ts

```diff
@@ -157,7 +157,7 @@ class Representation {
         if (initDataArr === undefined || initDataArr.length === 0) {
           return acc;
         }
-        const initData = concat(...initDataArr.map(({ data }) => data));
```

Comment:
So we only do this when the PSSH are found in the Manifest?

Not when they are found through an encrypted event?